name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DreiBankenEDVPDFExtractor.java

```diff
@@ -0,0 +1,260 @@
+            .match("(?<nameContinued>.*)")
+            .assign((t, v) -> {
+                v.put("isin", v.get("isin"));
+                v.put("name", v.get("name"));                            
+                t.setSecurity(getOrCreateSecurity(v));
+                t.setShares(asShares(v.get("shares")));
+            })
+                            
+            // Handelszeitpunkt: 04.01.2021 12:05:55
+            .section("date", "time")
+            .match("^(Handelszeitpunkt:).*(?<date>\\d+.\\d+.\\d{4}+) (?<time>\\d+:\\d+:\\d+).*")
+            .assign((t, v) -> {
+                if (v.get("time") != null)
+                    t.setDate(asDate(v.get("date"), v.get("time")));
+                else
+                    t.setDate(asDate(v.get("date")));
+            })
+            
+            // Wertpapierrechnung Wert 06.01.2021 EUR 205,30
+            .section("currency", "amount")
+            .match("^(Wertpapierrechn.* Wert) (\\d+.\\d+.\\d{4}+) (?<currency>\\w{3}+) *(?<amount>[\\d.-]+,\\d+).*")
+            .assign((t, v) -> {
+                t.setAmount(asAmount(v.get("amount")));
+                t.setCurrencyCode(v.get("currency"));
+            })
+    
+            // Kursgewinn-KESt EUR                -3,37
+            .section("tax", "currency").optional()
+            .match("^(Kursgewinn-KESt) (?<currency>\\w{3}).*(?<tax>-[\\d.]+,\\d{2})")
+            .assign((t, v) -> t.getPortfolioTransaction().addUnit(new Unit(Unit.Type.TAX,
+                            Money.of(asCurrencyCode(v.get("currency")), asAmount(v.get("tax"))))))
+            
+            // Dritt- und Börsengebühr EUR                0,02
+            .section("fee", "currency").optional() //
+            .match("^(Dritt.*B.*sengeb.*) (?<currency>\\w{3}+).*(?<fee>[\\d.-]+,\\d+).*")
+            .assign((t, v) -> t.getPortfolioTransaction()
+                            .addUnit(new Unit(Unit.Type.FEE,
+                                            Money.of(asCurrencyCode(v.get("currency")),
+                                                            asAmount(v.get("fee"))))))
+    
+            .wrap(BuySellEntryItem::new);
+    }
+    
+    @SuppressWarnings("nls")
+    private void addDividendeTransaction()
+    {
+        DocumentType newType = new DocumentType(".*(Ausschüttung|Dividende).*");
+        this.addDocumentTyp(newType);
+
+        Block block = new Block(".*(Ausschüttung|Dividende).*");
+        newType.addBlock(block);
+        Transaction<AccountTransaction> pdfTransaction = new Transaction<AccountTransaction>()
+
+        .subject(() -> {
+            AccountTransaction entry = new AccountTransaction();
+            entry.setType(AccountTransaction.Type.DIVIDENDS);
+            return entry;
+        });
+
+        Block firstRelevantLine = new Block(".*(Kauf|Verkauf).*");
+        newType.addBlock(firstRelevantLine);
+        firstRelevantLine.set(pdfTransaction);
```

Comment:
I think that is a left over, right? ```Kauf|Verkauf``` is never matched in the document type because it looks for ```Ausschüttung/Dividende``` above. I removed this.